sci-physics/vmc/vmc-1.0_p2.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+if [[ ${PV} == *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/vmc-project/${PN}.git"
+else
+	MY_PV=$(ver_rs 1-2 -)
+	SRC_URI="https://github.com/vmc-project/${PN}/archive/v${MY_PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+	S="${WORKDIR}/${PN}-${MY_PV}"
+fi
+
+DESCRIPTION="The Virtual Monte Carlo core library."
+HOMEPAGE="https://vmc-project.github.io/"
+
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="+c++11 c++14 c++17 doc"
+
+REQUIRED_USE="^^ ( c++11 c++14 c++17 )"
+
+RDEPEND="c++11? (
+		>=sci-physics/root-6.18:=[c++11,-vmc]
+	)
+	c++14? (
+		>=sci-physics/root-6.18:=[c++14,-vmc]
+	)
+	c++17? (
+		>=sci-physics/root-6.18:=[c++17,-vmc]
+	)"
+DEPEND="${RDEPEND}
+	doc? ( app-doc/doxygen[dot] )"
+
+DOCS=(
```

Comment:
Maybe one line.

Reply:
Thanks, adapted! Will also perform this change for my other PRs as applicable.